### ISSUE TITLE
Missing Accessibility Features

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#20]
Hi there! I really appreciate the work done on this project. I just wanted to kindly point out that it seems to be missing some accessibility features. It would be great to have better support for all users, including those who rely on assistive technologies. Thanks for considering this!
